### ISSUE TITLE
Australian $.  I am assuming the register supports it; any idea when square register sdk will do ?  tx.

### ISSUE THREAD

**Issue by raytetlow** [square/SquareRegisterSDK-iOS#4]
aus$ - I am assuming the register app now supports it; any idea when square register sdk support it ?  and I am correct in saying that USD, CAD and YEN are all sdk supported currencies ?  thanks.

**Comment by tadeegan** [comment]
Judging by this method it seems only USD CAD and JPY are supported:
```

+ (nonnull NSSet *)_setOfSupportedISO4217CurrencyCodes;
{
    static NSSet *setOfSupportedISO4217CurrencyCodes = nil;
    static dispatch_once_t onceToken;
    dispatch_once(&onceToken, ^{
        setOfSupportedISO4217CurrencyCodes = [NSSet setWithObjects:@"USD", @"CAD", @"JPY", nil];
    });
    return setOfSupportedISO4217CurrencyCodes;
}
```

**Comment by kuler90** [comment]
What's next? When it will be added?

**Comment by jhankin** [comment]
I'm pleased to announce that AUD is now available in SquareRegisterSDK-iOS 2.0.  Just update your installation to the latest version and you'll see that the AUD currency code is now in the set of supported currency codes.  Thanks for your patience!
